Collections/Map Utilities/map.alias

```diff
@@ -1207,59 +1219,63 @@ if c:
      break
    if not foundBattle:
     desc.append(f"Did not find the battle you were looking for")
+  #If user wants to load a saved battle
   if args.last('loadbattle'):
    argsBattle = args.last('loadbattle').lower()
    foundBattle = ""
    for searchBattle in userBattles:
     if argsBattle in searchBattle.lower():
      foundBattle = userBattles[searchBattle]
+     foundBattleName = searchBattle
      for searchCombatant in foundBattle:
       if searchCombatant not in ("mapattach"):
        foundCombatants = dict(list(foundBattle.items())[1:])
        currentCombatants = [combat().get_combatant(name)]
+       #Here saved combatant notes are loaded unless combatant isn't in combat
        if all([combat().get_combatant(searchCombatant) for searchCombatant in foundCombatants.keys()]):
         out = dict(list(foundBattle.items())[1:])
-        foundMapload = foundBattle["mapattach"]
-        if "mapbg" in foundMapload:
-         mapbg = foundMapload["mapbg"]
-        else:
-         mapbg = ""
-        if "mapsize"  in foundMapload:
-         mapsize = foundMapload["mapsize"]
-        else:
-         mapsize = None
-        if "mapoptions" in foundMapload:
-         mapoptions = foundMapload["mapoptions"]
-        else:
-         mapoptions = ""
-        if "walls" in foundMapload:
-         walls = foundMapload["walls"]
-        else:
-         walls = []
-        if "objects" in foundMapload:
-         objects = foundMapload["objects"]
-        else:
-         objects = []
-        if "json" in foundMapload:
-         loadedjson = foundMapload["json"].split(", ")
-        else:
-         loadedjson = []
-        if "view" in foundMapload:
-         mapviewlocation, mapviewsize = foundMapload["view"].replace('::',':').split(':')
-        desc.append(f"Loaded the battle `{argsBattle}`")
-        break
        else:
+        #Here lets user know what combatants they need to add to load ther info
```

Comment:
Below was my fix for the -loadbattle bug. It will load all the mapinfo but if there was a missing combatant that was saved in the battle, then it won't load the combatants notes until all combatants are in combat.